lib/src/components/modal/modal.scss

```diff
@@ -83,7 +83,7 @@ $modal-small-width: 400px !default;
 }
 
 .react-modal-header {
-  @apply flex flex-row w-full items-center relative h-16;
```

Comment:
Fixed heights in UI components are never ideal. This is causing issues in various headers in the app when they overflow with the screen height.